src/style.css

```diff
@@ -1,3 +1,97 @@
-body {
+:root {
+  --background-color: #fff;
+  --main-color: #00acdc;
+  --border-height: 3px;
+  --font-size: 12px;
+}
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Roboto', sans-serif;
+}
+
+.menu {
+  width: 100%;
+  background-color: var(--background-color);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.logo {
+  margin: 20px;
```

Comment:
Be careful with fixing sizes like `max-height` on elements, unless there is a specific requirement. This can cause issues with responsiveness or content overflow.